resources/production/chainspec.toml

```diff
@@ -70,6 +70,8 @@ max_associated_keys = 100
 max_runtime_call_stack_height = 12
 # Minimum allowed delegation amount in motes
 minimum_delegation_amount = 500_000_000_000
+# Global state prune batch size (0 = this feature is off)
+prune_batch_size = 50
```

Comment:
Don't we want to reset it to zero in dev already? So pruning is effectively doing nothing

Reply:
Good point.  Done in [cd55241](https://github.com/casper-network/casper-node/pull/3889/commits/cd552410b6f0c305ef75bb82cfdc6eacfc0ab3fd).